samples/nrf9160/modem_shell/src/link/link.c

```diff
@@ -273,6 +273,21 @@ static int link_default_pdp_context_auth_set(void)
 	return 0;
 }
 
+static int link_enable_rel14_features(void)
+{
+	static const char rel14feat_at_cmd[] = "AT%REL14FEAT=1,1,1,1,1";
+	char response[MOSH_AT_CMD_RESPONSE_MAX_LEN + 1];
+
+	if (at_cmd_write(rel14feat_at_cmd, response,
+				sizeof(response), NULL) != 0) {
```

Comment:
The convention has been that parameters would be aligned to start at the same level, i.e. the second line here should start at the same level as the first parameter for the function (the fine tuning usually has to be done with spaces).